src/main/io/osd.c

```diff
@@ -812,40 +832,68 @@ static void osdShowStats(void)
     displayClearScreen(osdDisplayPort);
     displayWrite(osdDisplayPort, 2, top++, "  --- STATS ---");
 
-    if (STATE(GPS_FIX)) {
+    if (osdConfig()->enabled_stats[OSD_STAT_ARMEDTIME]) {
+        displayWrite(osdDisplayPort, 2, top, "ARMED TIME       :");
+        tfp_sprintf(buff, "%02d:%02d", stats.armed_time / 60, stats.armed_time % 60);
+        displayWrite(osdDisplayPort, 22, top++, buff);
+    }
+
+    if (osdConfig()->enabled_stats[OSD_STAT_FLYTIME]) {
+        displayWrite(osdDisplayPort, 2, top, "FLY TIME         :");
+        tfp_sprintf(buff, "%02d:%02d", flyTime / 60, flyTime % 60);
+        displayWrite(osdDisplayPort, 22, top++, buff);
+    }
+
+    if (osdConfig()->enabled_stats[OSD_STAT_MAX_SPEED] && STATE(GPS_FIX)) {
         displayWrite(osdDisplayPort, 2, top, "MAX SPEED        :");
         itoa(stats.max_speed, buff, 10);
         displayWrite(osdDisplayPort, 22, top++, buff);
     }
 
-    displayWrite(osdDisplayPort, 2, top, "MIN BATTERY      :");
-    tfp_sprintf(buff, "%d.%1dV", stats.min_voltage / 10, stats.min_voltage % 10);
-    displayWrite(osdDisplayPort, 22, top++, buff);
-
-    displayWrite(osdDisplayPort, 2, top, "MIN RSSI         :");
-    itoa(stats.min_rssi, buff, 10);
-    strcat(buff, "%");
-    displayWrite(osdDisplayPort, 22, top++, buff);
+    if (osdConfig()->enabled_stats[OSD_STAT_MIN_BATTERY]) {
+        displayWrite(osdDisplayPort, 2, top, "MIN BATTERY      :");
```

Comment:
Yes, "before" is `b1e6a8b` and "after" is `fe9689f`.
That also includes the three new statistic options added in this PR.